pageserver/src/disk_usage_eviction_task.rs

```diff
@@ -122,6 +121,10 @@ async fn disk_usage_eviction_task(
     tenants_dir: &Path,
     cancel: CancellationToken,
 ) {
+    scopeguard::defer! {
+        info!("disk usage based eviction task finishing");
+    };
+
```

Comment:
If this is to be added then the other info!'s about shutdown should be removed.

On shutdown this would give us two messages.

Reply:
What other messages? You mean the "shutting down" ones?

Reply:
Line L170R173: `info!("shutting down");`

Reply:
Line L135R138: `info!("shutting down");`